p2p/src/payment_channel.rs

```diff
@@ -476,8 +476,11 @@ impl PaymentChannels {
 mod test {
     use crate::{ContactInfo, PaymentChannel};
     use blake2::Blake2b512;
+    use circuits::*;
```

Comment:
Again, ideally, avoid depending on `circuits` here. PaymentChannel doesn't care about what data it has, it's just a vessel.